nautobot/ipam/tests/test_views.py

```diff
@@ -1084,13 +1084,29 @@ def setUpTestData(cls):
         }
 
         cls.bulk_edit_data = {
-            "vlan_group": vlangroups[0].pk,
+            "vlan_group": cls.vlangroups[0].pk,
             "tenant": Tenant.objects.first().pk,
             "status": status.pk,
             "role": roles[0].pk,
             "description": "New description",
         }
 
+    def test_vlan_group_not_belong_to_vlan_locations(self):
+        """Test that a VLAN cannot be assigned to a VLAN Group that is not in the same location as the VLAN."""
+        vlan_group = self.vlangroups[0]
+        form_data = self.form_data.copy()
+        form_data["vlan_group"] = vlan_group.pk
+        form_data["locations"] = [self.locations.last().pk]
+        self.add_permissions("ipam.add_vlan")
```

Comment:
Do we also need to add `view_vlangroup` permissions? Or not? I'm bugged by the fact that this test was passing even when incorrectly using `self.form_data` below. Makes me think it's not testing what it thinks it is.